File: Modul 6/Soal 5/PRAK605-NIM-Nama.c
#include <stdio.h>

void multiply(int n, int MatxA[][n], int MatxB[][n], int result[][n]){
    for(int i = 0; i < n; i++){
        for(int j = 0; j < n; j++){
            result[i][j] = 0;
            for(int k = 0; k < n; k++){
                result[i][j] += MatxA[i][k] * MatxB[k][j];
            }
        }
    }
}

int main(){
    int n;
    
    scanf("%d", &n);

    int MatxA[n][n], MatxB[n][n], result[n][n];

    printf("Matriks A\n");
    for(int i = 0; i < n; i++){
        for(int j = 0; j < n; j++){
            scanf("%d", &MatxA[i][j]);
        }
    }

    printf("Matriks B\n");
    for(int i = 0; i < n; i++){
        for(int j = 0; j < n; j++){
            scanf("%d", &MatxB[i][j]);
        }
    }

    multiply(n, MatxA, MatxB, result);

    printf("Matriks AXB\n");
    for(int i = 0; i < n; i++){
        for(int j = 0; j < n; j++){
            printf("%d ", result[i][j]);
        }
        printf("\n");
    }
    return 0;
}

File: Modul 6/Soal 5/PRAK605-NIM-Nama.py
def multiply(n, MatxA, MatxB):
    result = [[0] * n for _ in range(n)]
    for i in range(n):
        for j in range(n):
            for k in range(n):
                result[i][j] += MatxA[i][k] * MatxB[k][j]
    return result

n = int(input())

print("Matriks A")
MatxA = [list(map(int, input().split())) for _ in range(n)]

print("Matriks B")
MatxB = [list(map(int, input().split())) for _ in range(n)]

result = multiply(n, MatxA, MatxB)

print("Matriks AXB")
for row in result:
    print(" ".join(map(str, row)))